public/main.js

```diff
@@ -75,36 +89,54 @@ function reloadTodoList() {
     while (todoList.firstChild) {
         todoList.removeChild(todoList.firstChild);
     }
+    deleteCompletedBtn.style.display = "none";
     todoListPlaceholder.style.display = "block";
     getTodoList(function(todos) {
+        //total number of todos which have been set as complete
+        var completedTodos = todos.filter(function(todo) {
```

Comment:
Rename to completedTodosCount or totalNumbersOfCompletedTodos or otherwise, and remove comment.